docs/docs/examples/property_graph/property_graph_basic.ipynb

```diff
@@ -0,0 +1,415 @@
+{
```

Comment:
<p>this is great </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/13747/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>